jenkins.io.tf

```
@@ -0,0 +1,20 @@
+resource "azurerm_resource_group" "jenkins_io" {
+  name     = "jenkinsio"
+  location = var.location
+}
+
+resource "azurerm_storage_account" "jenkins_io" {
+  name                     = "jenkinsio"
+  resource_group_name      = azurerm_resource_group.jenkins_io.name
+  location                 = azurerm_resource_group.jenkins_io.location
+  account_tier             = "Premium"
+  account_replication_type = "LRS"
```

```suggestion
  account_replication_type = "ZRS"
```

As we need to be able to spin up VMs (or AKS nodes) on different availability zones tro reach this file storage.